docs/static/security/grant-access.asciidoc

```diff
@@ -13,24 +13,30 @@ privileges  for the Logstash indices. You can create roles from the
 ---------------------------------------------------------------
 POST _security/role/logstash_reader
 {
-  "cluster": ["manage_logstash_pipelines"]
+  "cluster": ["manage_logstash_pipelines"],
+  "indices": [
+    {
+      "names": [ "logstash-*" ],
+      "privileges": ["read","view_index_metadata"]
+    }
+  ]
 }
 ---------------------------------------------------------------
 
 . Assign your Logstash users the `logstash_reader` role. If the Logstash user
 will be using
 {logstash-ref}/logstash-centralized-pipeline-management.html[centralized pipeline management],
-also assign the `logstash_admin` role. You can create and manage users from the
+also assign the `logstash_system` role. You can create and manage users from the
```

Comment:
we need cluster privilege "monitor" to query `/_xpack` when using central pipeline management.

Elasticsearch has two built-in roles: logstash_admin, logstash_system
logstash_admin
```
{
  "cluster": ["manage_logstash_pipelines"], 
  "indices": [
    {
      "names": [ ".logstash*" ], 
      "privileges": ["create","read","delete","index","manage"]  
    }
  ]
}
```


logstash_system
```
{
  "cluster": ["monitor", "cluster:admin/xpack/monitoring/bulk"]
}
```

So, we need logstash_system here